overview/glossary.markdown

```diff
@@ -41,74 +102,177 @@ Used to restrict when / where promises are evaluated.
+
+#### GPL3
+
+The GNU Public License, version 3.
+
+#### Graphical user interface (GUI)
+
+As opposed to text / command line based interfaces, GUIs use icons, images, color, spacing, and more complex layouts to improve the user experience.
+
+The CFEngine GUI is called Mission Portal and is accessible via a web browser.
+It shows you useful information about your infrastructure, and provides easy ways to make changes.
+
 #### Host
 
-Unix terminology for a computer that runs "guest programs."
-In practice, "host" is a synonym for "computer."
+UNIX terminology for a computer the runs _guest programs_.
+In practice, _host_ is a synonym for _computer_.
+
+In CFEngine, all machines (physical or virtual) which have an installation of CFEngine are considered _hosts_.
+We split them into 2 roles (categories) - hubs and clients.
 
 #### Hub
 
-A software component in CFEngine Enterprise that acts as a single point of management in a local "star-network."
-The term "hub" is sometimes used to mean policy distribution server, but more commonly a running `cf-hub` process that does report collection from all CFEngine managed hosts.
-The term hub means the centre of a wheel, from which multiple spokes emerge.
+The term hub means the center of a wheel, from which multiple spokes emerge.
+
+In CFEngine, the hub is the host responsible for collecting reports from hosts and serving them policy.
+In addition to the components installed on other CFEngine hosts (clients), the hub runs a database (PostgreSQL), a web server (Apache) and a few additional CFEngine components, most notably `cf-hub` which connects to hosts and retrieves their reporting data.
+
+Due to the multiple purposes this host serves, it is sometimes referred to as the policy server, the reporting hub, or the report collector.
+In typical CFEngine Enterprise setups, all hubs are policy servers and all policy servers are hubs, so the distinction is not so important.
+In general, hub is the preferred term to describe the role of what this host does, and which package to install on it.
+
+See CFEngine role.
+
+#### Lightweight directory access protocol (LDAP)
+
+A kind of _phone book_ service providing information about persons and computers in an organization.
+
+#### Libraries
+
+A library generally refers to collection of standardized CFEngine code that can be reused in different scenarios and environments.
+This might be bundles of promises, or reusable body-parts.
 
 #### Logs
-#### Loops
-#### Menus
-#### Mission Portal
-#### Monitoring
+
+Log files tell you some historic, usually timestamped, information about events which happened in the past.
+In CFEngine, there are a few notable log files:
+
+* `/var/logs/CFEngineInstall.log` - Information about the installation, especially useful if installing the package failed.
+* `/var/cfengine/outputs/` - Output logs of previous scheduled agent runs (if any).
```

Comment:
```suggestion
* `/var/cfengine/outputs/` - Output logs of previously scheduled agent runs (if any).
```

Reply:
This one is wrong, the 2 sentences have different meanings. We are talking about the previous agent runs, which were scheduled, not the agent runs which were previously scheduled.